src/mito2/src/sst/file.rs

```diff
@@ -137,6 +137,20 @@ impl FileHandle {
     pub fn mark_deleted(&self) {
         self.inner.deleted.store(true, Ordering::Relaxed);
     }
+
+    #[inline]
+    pub fn compacting(&self) -> bool {
+        self.inner.compacting.load(Ordering::Relaxed)
+    }
+
+    #[inline]
+    pub fn mark_compacting(&self, compacting: bool) {
```

Comment:
suggestion: `mark_xxx` usually sets the flag to true. This is a setter so let's use `set_compacting`.
```suggestion
    pub fn set_compacting(&self, compacting: bool) {
```